### PR TITLE
DOC-667 Timeplus output

### PR DESCRIPTION
**Pull request by asimms41**
## Description

Resolves: [DOC-667](https://redpandadata.atlassian.net/browse/DOC-667)
Review deadline: 30th October

Related PR to enable Timeplus output in the Cloud: https://github.com/redpanda-data/cloud-docs/pull/102

## Page previews

[timeplus output](https://deploy-preview-79--redpanda-connect.netlify.app/redpanda-connect/components/outputs/timeplus/)

## Checks

- [ ] New feature
- [x] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

[DOC-667]: https://redpandadata.atlassian.net/browse/DOC-667?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: modules/components/pages/outputs/timeplus.adoc [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+:page-beta: true
+:categories: ["Services"]
+
+// © 2024 Redpanda Data Inc.
+
+
+component_type_dropdown::[]
+
+
+Sends messages to a data stream on https://docs.timeplus.com/[Timeplus Enterprise (Cloud or Self-Hosted)^] using the https://docs.timeplus.com/ingest-api[Ingest API^], or directly to the `timeplusd` server process.
```
Comment (asimms41) [inline modules/components/pages/outputs/timeplus.adoc:13]: Is it correct to refer to timeplusd as a server process?

**Review by mihaitodor — APPROVED** [review]
LGTM, just a few nits.

File: modules/components/pages/outputs/timeplus.adoc [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+
+=== `username`
+
+The username for the Timeplus application server. This field is required if you are sending message to Timeplus Enterprise (Self-Hosted) or `timeplusd`.
+
+*Type*: `string`
+
+=== `password`
+
+The password for the Timeplus application server. This field is required if you are sending message to Timeplus Enterprise (Self-Hosted) or `timeplusd`.
```
[the remark this answers is not in the document]
Comment (asimms41) [reply modules/components/pages/outputs/timeplus.adoc]: Updated.

**Review by Jeffail — APPROVED** [review]

File: modules/components/pages/outputs/timeplus.adoc [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+
+--
+======
+
+
+=== Unstructured messages
+
+If your upstream data source or pipeline returns unstructured messages, such as strings, you can configure an output processor to transform them into a structured messages and then pass them to the output.
+
+This example creates structured messages using the `raw` field, and stores the original string content into it. You can modify the name of the field, but you must also add either the the `raw` field name or your custom field name to the destination data stream.
```
Comment (kbatuigas) [inline modules/components/pages/outputs/timeplus.adoc:150]: Is `raw` supposed to be one of the fields listed on this doc? And am I understanding correctly that `raw` is like a function that parses and transforms values stored in it?

**Review by kbatuigas — APPROVED** [review]
Looks good! Added a question in case it's something we can clarify for readers

File: modules/components/pages/outputs/timeplus.adoc (hunk 3 above)
Comment (asimms41) [reply modules/components/pages/outputs/timeplus.adoc:150]: No `raw` is just an example field name, and my understanding that yes `raw` is a Blobland function that parses and transforms the string value. I'll try to reword to make that clearer.

Comment (asimms41) [reply modules/components/pages/outputs/timeplus.adoc]: @kbatuigas, I've tried to reword the text explaining the example. Is it any better now?

Comment (kbatuigas) [reply modules/components/pages/outputs/timeplus.adoc]: @kbatuigas yes that looks good to me. Thank you!

**Pull request merged**